### ISSUE TITLE
Revise chapter 1, challenge 2 intro copy

### ISSUE THREAD

**Issue by GBKS** [saving-satoshi/saving-satoshi#183]
Do we need some tool tips with explainers for terms used in the chapter 1, challenge 2 intro page? Also, the last sentence of the intro page is identical to the copy on the challenge page. Probably best if @satsie has the final word on this copy, I think she had some comments on Figma. There is some revised copy in [Figma here](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=1604%3A19853&t=wtEGAyNRp9cwYerM-1).

![image](https://user-images.githubusercontent.com/695901/220908467-a4ba70cf-f563-403a-ac15-face15ee458d.png)

**Comment by satsie** [comment]
I think we can get away with no tool tips for Chapter 1, Challenge 2 (side note, what if we abbreviated these internally as something like "C1C2"?)

If my [revised copy and screens](https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=1965%3A21437&t=0Og2PDyNKdCHivdc-1) look good to you, I'd prefer if we can implement those. Otherwise, we can just go with the original set that is in the Chapter 1 demo specs.

**Comment by ecurrencyhodler** [comment]
If the revised copy/screens look good, stacie mentioned that it would also unblock https://github.com/saving-satoshi/saving-satoshi/issues/180 so we can go ahead and remove tips from C1C1 page 2.

**Comment by ecurrencyhodler** [comment]
I think christoph indicated in figma that we are g2g with the designs. I am going to close this issue as it is no longer needed.
